process/elasticproc/accounts/accountsProcessor_test.go

```diff
@@ -177,9 +177,10 @@ func TestGetESDTInfoNFTWithMetaData(t *testing.T) {
 	require.Equal(t, big.NewInt(1), balance)
 	require.Equal(t, "6f6b", prop)
 	require.Equal(t, &data.TokenMetaData{
-		Name:      nftName,
-		Creator:   creator,
-		Royalties: 2,
+		Name:       nftName,
+		Creator:    creator,
+		Royalties:  2,
+		Attributes: []byte{},
```

Comment:
`make([]byte, 0)` instead of literal declaration

Reply:
done